### ISSUE TITLE
Community Usage

### ISSUE THREAD

**Issue by peterdesio** [shunkosa/lightning-chatter-messenger#5]
@shunkosa 

I updated the component for use within a customer community. Everything looks good and functions properly as an internal user logged into the community, but when I log in as a customer community user I cannot send a message.

Profile has Create/Edit access to ChatterMessageEvent & the ChatterMessage class. Any ideas?

**Comment by shunkosa** [comment]
Thank you for the comment @peterdesio !
Are all the apex methods already modified to include the community id?

**Comment by jnat1108** [comment]
How can i  use this component in salesforce communities? 
Can you send me a step by step on that on jnat.developer@gmail.com.

Thanks

**Comment by shunkosa** [comment]
@jnat1108 Thank you for reaching out. Unfortunately, community is not supported yet. You need to edit Apex class for community.

**Comment by jnat1108** [comment]
Hi Shunkosa

Does it support auto refresh ????
I did not see message coming in automatically.
Please let me know.

Jayant

**Comment by jnat1108** [comment]
HI Shunkosa

Are you working on that bug?

**Comment by shunkosa** [comment]
@jnat1108 Not yet. It supports auto refresh by platform event. Could you please elaborate on the situation?

**Comment by jnat1108** [comment]
Yes but my component is not getting subscribed to the platform event.
So incoming messages are not being displayed.

Can you tell me what can be the cause ?
I  am using it in a community.

Jayant

**Comment by shunkosa** [comment]
Oh, I see. Unfortunately this repository doesn't support community now. It will but take time. Does any error message show on the browser console?
